eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -123,7 +124,9 @@ public int hashCode() {
         }
         this.terms.get().add(this.vertex);
         final List<String> list = new ArrayList<>(this.attrs.size());
-        for (final Map.Entry<String, Attr> ent : this.attrs.entrySet()) {
+        for (final Map.Entry<String, Attr> ent : this.attrs.entrySet().stream().filter(
+            e -> List.of("σ", "ρ").contains(e.getKey())
+        ).collect(Collectors.toList())) {
```

Comment:
@mximp this looks like a hack, I would not do this

Reply:
@yegor256 If we remove those then no parameters will be printed in the log at all. For debugging purposes it's useful to see the chain.